moveit_kinematics/cached_ik_kinematics_plugin/README.md

```diff
@@ -54,7 +54,7 @@ Below is a complete list of all arguments:
 
 ## Advanced Usage: Creating Wrappers for Other IK Solvers
 
-The Cached IK Kinematics Plugin is implemented as a wrapper around classed derived from the [`kinematics::KinematicsBase` abstract base class](http://docs.ros.org/latest-lts/api/moveit_core/html/classkinematics_1_1KinematicsBase.html). Wrappers for the `kdl_kinematics_plugin::KDLKinematicsPlugin` and `srv_kinematics_plugin::SrvKinematicsPlugin` classes are already included in the plugin. For any other solver, you can create a new kinematics plugin. The C++ code for doing so is extremely simple; here is the code to create a wrapper for the KDL solver:
+The Cached IK Kinematics Plugin is implemented as a wrapper around classed derived from the `kinematics::KinematicsBase` abstract base class. Wrappers for the `kdl_kinematics_plugin::KDLKinematicsPlugin` and `srv_kinematics_plugin::SrvKinematicsPlugin` classes are already included in the plugin. For any other solver, you can create a new kinematics plugin. The C++ code for doing so is extremely simple; here is the code to create a wrapper for the KDL solver:
```

Comment:
Couldn't find an up-to-date link for this. Most recent I could find is from [lunar](http://docs.ros.org/en/lunar/api/moveit_core/html/classkinematics_1_1KinematicsBase.html)

Is there another place that documentation has moved for this that could be linked here?

Reply:
There should be a noetic link for this.